src/components/DirectMessage/Friends.tsx

```diff
@@ -67,12 +69,15 @@ const Friends: React.FC<FriendsProps> = ({ friends, fetchFriends }) => {
 
   const handleRemoveFriend = async () => {
     try {
+      setIsConfirmModalButtonDisable(true);
       await axiosInstance.patch(`/friends/${openOptions}/rejected`);
       setOpenOptions(null);
       fetchFriends(view);
       showSnackbar('Friend removed Succesfully!', 'success');
+      setIsConfirmModalButtonDisable(false);
     } catch (error) {
       showSnackbar('An error occurred. Please try again.', 'error');
+      setIsConfirmModalButtonDisable(false);
```

Comment:
instead of setting it twice in try and catch both, use finally to set it to false